explorer/interpreter/type_checker.cpp

```diff
@@ -2661,7 +2705,8 @@ auto TypeChecker::CheckAddrMeAccess(
     access->set_is_addr_me_method();
     CARBON_ASSIGN_OR_RETURN(
         Nonnull<const Value*> me_type,
-        Substitute(bindings, &func_decl->self_pattern().static_type()));
+        Substitute(bindings, &func_decl->self_pattern().static_type(),
+                   func_decl->source_loc()));
```

Comment:
```suggestion
                   access->source_loc()));
```
I think we want the location of the call expression here, rather than the location of the callee.